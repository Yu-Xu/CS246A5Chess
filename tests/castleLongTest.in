game human human
move d2 d4
move d7 d5
move c1 e3
move c8 e6
move b1 c3
move b8 c6
move d1 d2
move d8 d7
move e1 c1
move e8 c8
