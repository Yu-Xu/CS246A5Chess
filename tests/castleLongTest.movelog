8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ P _ _
3 _ _ _ _ 
2 PPP_PPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 ppp pppp
6  _ _ _ _
5 _ _p_ _ 
4  _ P _ _
3 _ _ _ _ 
2 PPP_PPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 ppp pppp
6  _ _ _ _
5 _ _p_ _ 
4  _ P _ _
3 _ _ B _ 
2 PPP_PPPP
1 RN_QKBNR
  abcdefgh
8 rn qkbnr
7 ppp pppp
6  _ _b_ _
5 _ _p_ _ 
4  _ P _ _
3 _ _ B _ 
2 PPP_PPPP
1 RN_QKBNR
  abcdefgh
8 rn qkbnr
7 ppp pppp
6  _ _b_ _
5 _ _p_ _ 
4  _ P _ _
3 _ N B _ 
2 PPP_PPPP
1 R _QKBNR
  abcdefgh
8 r_ qkbnr
7 ppp pppp
6  _n_b_ _
5 _ _p_ _ 
4  _ P _ _
3 _ N B _ 
2 PPP_PPPP
1 R _QKBNR
  abcdefgh
8 r_ qkbnr
7 ppp pppp
6  _n_b_ _
5 _ _p_ _ 
4  _ P _ _
3 _ N B _ 
2 PPPQPPPP
1 R _ KBNR
  abcdefgh
8 r_ _kbnr
7 pppqpppp
6  _n_b_ _
5 _ _p_ _ 
4  _ P _ _
3 _ N B _ 
2 PPPQPPPP
1 R _ KBNR
  abcdefgh
8 r_ _kbnr
7 pppqpppp
6  _n_b_ _
5 _ _p_ _ 
4  _ P _ _
3 _ N B _ 
2 PPPQPPPP
1 _ KR_BNR
  abcdefgh
8  _kr bnr
7 pppqpppp
6  _n_b_ _
5 _ _p_ _ 
4  _ P _ _
3 _ N B _ 
2 PPPQPPPP
1 _ KR_BNR
  abcdefgh
