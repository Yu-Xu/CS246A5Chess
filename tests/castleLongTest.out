8  _kr bnr
7 pppqpppp
6  _n_b_ _
5 _ _p_ _ 
4  _ P _ _
3 _ N B _ 
2 PPPQPPPP
1 _ KR_BNR
